config/locales/en.yml

```diff
@@ -47,13 +47,15 @@ en:
       trips: "Trips"
   families:
     index:
-      title: "Families"
       add_family: "Add a Family"
-      destroy: "Destroy"
-      edit: "Update"
-      submit: "Add"
       destroy: "Delete"
       edit: "Edit"
+      active_families: "Families Active"
+      not_active_families: "Families Not Active"
```

Comment:
Similar here: it reads better as "Inactive families"

Reply:
Done in sha `3e97938`